doc/classes/TileSet.xml

```diff
@@ -34,6 +22,8 @@
 			<argument index="3" name="tile_location" type="Vector2">
 			</argument>
 			<description>
+				Forwards the subtile selection. Overriding this method allows you to create custom logic to select wich subtile to use when painting. Works with [constant AUTO_TILE] and [constant ATLAS_TILE], but when it is an atlas, [code]bitmask[/code] will always be [constant Vector2.ZERO].
```

Comment:
Not sure if I should use `Vector2.ZERO` or `Vector2.INF`. Using `Vector2.INF` may be good to identify that is an atlas.